### PR TITLE
go: add French translation

### PR DESCRIPTION
**Pull request by melvinxeu**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/go.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# go
+
+> Outil de gestion du code source du go.
```
Comment (nicokosi) [inline pages.fr/common/go.md:3]: I propose this slight update (IMHO, it removes the ambiguity with [go, the game](https://en.wikipedia.org/wiki/Go_(game))):
```suggestion
> Outil de gestion du code source Go.
```

File: pages.fr/common/go.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> Certaines sous-commandes telles que `go build` ont leur propre documentation d'utilisation.
+> Plus d'informations: <https://golang.org>.
```
Comment (nicokosi) [inline pages.fr/common/go.md:5]:
```suggestion
> Plus d'informations : <https://golang.org>.
```

File: pages.fr/common/go.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Télécharger et installer un paquet, spécifié par son chemin d'importation:
```
[the remark this answers is not in the document]
Comment (nicokosi) [reply pages.fr/common/go.md:6]: (and so on :wink:)

**Comment by navarroaxel** [comment]
Hey @melvinxeu, did you see the review?

File: pages.fr/common/go.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`go get {{chemin_du_paquet}}`
+
+- Compiler et exécuter un fichier source (il doit contenir un paquet `main`):
```
Comment (navarroaxel) [inline pages.fr/common/go.md:10]:
```suggestion
- Compiler et exécuter un fichier source (il doit contenir un paquet `main`) :
```

File: pages.fr/common/go.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`go run {{fichier}}.go`
+
+- Compiler un fichier source dans un exécutable nommé:
```
Comment (navarroaxel) [inline pages.fr/common/go.md:14]:
```suggestion
- Compiler un fichier source dans un exécutable nommé :
```

File: pages.fr/common/go.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`go build -o {{executable}} {{fichier}}.go`
+
+- Compile le paquet présent dans le répertoire courant:
```
Comment (navarroaxel) [inline pages.fr/common/go.md:18]:
```suggestion
- Compile le paquet présent dans le répertoire courant :
```

File: pages.fr/common/go.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`go build`
+
+- Exécuter tous les cas de test du paquet courant (les fichiers doivent se terminer par `_test.go`):
```
Comment (navarroaxel) [inline pages.fr/common/go.md:22]:
```suggestion
- Exécuter tous les cas de test du paquet courant (les fichiers doivent se terminer par `_test.go`) :
```

File: pages.fr/common/go.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`go test`
+
+- Compiler et installer le paquet actuel:
```
Comment (navarroaxel) [inline pages.fr/common/go.md:26]:
```suggestion
- Compiler et installer le paquet actuel :
```

File: pages.fr/common/go.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`go install`
+
+- Initialiser un nouveau module dans le répertoire courant:
```
Comment (navarroaxel) [inline pages.fr/common/go.md:30]:
```suggestion
- Initialiser un nouveau module dans le répertoire courant :
```

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**